features/support/object_model_pages/ivl/plan_shopping/ivl_plan_shopping.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+#Ivl plan shopping
+class IvlPlanShopping
+  def self.premium_amount_from_filter
+    "input[type='text'][class='plan-metal-premium-from-selection-filter form-control']"
+  end
+
+  def self.premium_amount_to_filter
+    "input[type='text'][class='plan-metal-premium-to-selection-filter form-control fr']"
+  end
+
+  def self.plans_count
+    "Plans:   4"
```

Comment:
please add a `data-cuke=""` selector instead of selecting hard-coded english text

Reply:
@markgoho In this case, we want to make sure that the page is displaying 4 plans